contracts/contracts/Ramp.sol

```diff
@@ -105,6 +109,12 @@ contract Ramp is Ownable {
         mapping(bytes32 => bool) isDenied;  // Mapping of venmoIdHash to boolean indicating if the user is denied
     }
 
+    struct GlobalAccountInfo {
```

Comment:
Could merge `GlobalAccountInfo` and `AccountInfo`.  Both of them store state corresponding to a Venmo Id (in future other payment provider Id). Or if you want to keep them separate, rename this one to `AccountRampInfo` and `AccountInfo` to `AccountDepositInfo`.

Reply:
Hmm Account is tied to the Ethereum address while global account is tied to the venmoId since we have different permissions. Basically each Ethereum address can have 5 deposits. The deny list, current intent hash, and last on ramp timestamp are all enforced on a per venmo account basis

Reply:
Agreed on not loving the naming tho, I wanna try to think of something else. Maybe just `Account` and `SubAccount` because each Ethereum address is essentially a subaccount. But doing it that way kinda seems like you'd want to track all subaccounts within tied to a venmoId. Don't really want to / think it's necessary to manage that state tho.

Reply:
Adding the comment makes it a lot more clear. We can roll with this for now, and rethink types of accounts and their relations in the next iteration.